django/db/models.py

```diff
@@ -775,15 +959,16 @@ def get_properties(obj):
     """
     Custom method to grab all of the fields _and_ properties on model instances
     """
-    
-    attrs     = {}
+    attrs = {}
     MethodWrapperType = type(object().__hash__)
 
     for slot in dir(obj):
         try:
             attr = getattr(obj, slot)
 
-            if ( slot.find('_') == 0 or slot == '__dict__' or slot == '__class__' 
+            if ( slot.find('_') == 0 
```

Comment:
Suggest the following changes here:
* Just check `slot[0] == '_'` instead of using find.
* Double-underscore *anything* should probably be dropped, right? And regardless, doesn't the first test for underscore subsume all of the explicit comparisons to \_\_dict\_\_ etc?
* Remember that `isinstance` can take a tuple of types as the second arg, so the last bit could be simplified to:
  `isinstance(attr, (MethodWrapperType, types.BuiltinMethodType, types.MethodType, ...))`